docs/content/when-to-use-signals.md

```diff
@@ -12,5 +12,4 @@ tags:
 - Signals can receive data, but cannot return it.
 - When a Signal is received for a running Workflow, Temporal persists the Signal event and payload in the Workflow history.
   The Workflow can then process the Signal at any time afterwards without the risk of losing the information.
-- In the Go SDK, a Workflow can pause until it receives a Signal by blocking on a **Signal channel**.
```

Comment:
@flossypurse this is coming up again

Reply:
I want to make sure that info is retained in the appropriate place.
I have another PR up to refactor Signal docs https://github.com/temporalio/documentation/pull/711 - which can include addressing this 
So, maybe just leave it in for the moment and I will prioritize addressing this.